spring-batch-docs/asciidoc/step.adoc

```diff
@@ -944,14 +948,14 @@ public interface ItemWriteListener<S> extends StepListener {
 }
 ----
 
-The `beforeWrite` method will be called
+The `beforeWrite` method is called
         before `write` on the
-        `ItemWriter`, and is handed the item that will be
-        written. The `afterWrite` method will be called
+        `ItemWriter` and is handed the item that is
```

Comment:
"...is handed the list of items to be written."  See https://docs.spring.io/spring-batch/trunk/apidocs/org/springframework/batch/core/ItemWriteListener.html#afterWrite-java.util.List-